src/Entities/Comment.php

```diff
@@ -8,13 +8,16 @@
     public string $content;
     public string $timeStamp;
 
-    public function __construct(int $id, int $authorid, int $blogid, string $content, string $timeStamp)
+    public string $username;
+
+    public function __construct(int $id, int $authorid, int $blogid, string $content, string $timeStamp, string $username)
     {
         $this->id = $id;
         $this->authorid = $authorid;
         $this->blogid = $blogid;
         $this->content = $content;
         $this->timeStamp = gmdate("d/m/y", strtotime($timeStamp));
```

Comment:
Can we get a unit test on this please, just to make sure that the timeStamp format works correctly. Just like you did with the blog test

Reply:
done